### PR TITLE
Adding trouble shooting guide for GSG

### PR DESCRIPTION
**Pull request by JanLin**
While I was trouble shooting the problem with running GSG and error 307 I saw others on rocket.chat report the same problem. The proposed fix did not work but after some investigation I was able to get the demo running smoothly. Hope the trouble shooting can help others that may come across similar problems.

### CODE REVIEW

**Comment by JanLin** [comment]
Signed-off-by: Jan Lindquist <jan.lindquist@dativa.com>

File: doc/getting-started/Trouble_shoot_GSG.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# TROUBLE SHOOTING Getting - Started Guide (GSG)
```
[the remark this answers is not in the document]
Comment (jovfer) [reply doc/getting-started/Trouble_shoot_GSG.md]: If we decide to keep this as separate file, I suggest to add links to it from
https://github.com/hyperledger/indy-sdk/blob/master/doc/getting-started/run-getting-started.md
and https://github.com/hyperledger/indy-sdk/blob/master/doc/getting-started/getting-started.md#explore-the-code

[the remark this answers is not in the document]
Comment (JanLin) [reply doc/getting-started/Trouble_shoot_GSG.md]: Good suggestion. I will add a links.

[the remark this answers is not in the document]
Comment (JanLin) [reply doc/getting-started/Trouble_shoot_GSG.md]: I have now made changes to the 2 files with new links.

**Pull request merged**